### PR TITLE
Update output stream cython declaration

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @aaltay Cython change got left out in https://github.com/apache/beam/pull/2081 PTAL

**Comment by dhalperi** [comment]
Is it possible to add new precommit testing to this PR so that the issue in #2081 is caught before merging?

**Comment by vikkyrk** [comment]
Updated `tox` config to enable them, was easier than I thought. Hoping everything works.

**Comment by vikkyrk** [comment]
Retest this please :)

**Comment by robertwb** [comment]
LTGM, and thanks for adding the tests toos.

**Comment by vikkyrk** [comment]
Tests are not getting triggered for some reason, @jasonkuster will be taking a look soon.

**Comment by dhalperi** [comment]
retest this please

(seems like Maven is partially down. Maybe it's related to AWS outage, or something else?)

**Comment by dhalperi** [comment]
retest this please

please work, internet.

**Comment by robertwb** [comment]
Tests look good. Merging.

File: sdks/python/tox.ini [hunk 1]
```diff
@@ -34,6 +34,26 @@ commands =
   python setup.py test
 passenv = TRAVIS*
 
+[testenv:py27cython]
```
Comment (aaltay) [inline sdks/python/tox.ini]: Please add `whitelist_externals=find` here, otherwise there will be a warning in the output.

**Pull request closed without merging**